.tests/integration/configs/fastp_merge_config.yaml

```diff
@@ -3,7 +3,7 @@ units: "units.tsv"
 
```

Comment:
Should we remove this since it's already defined in the first config?

Reply:
I updated it and removed anything unnecessary